Uno.Gallery/Uno.Gallery.UWP/Views/SamplePages/SwipeControlSamplePage.xaml

```diff
@@ -0,0 +1,186 @@
+
+									<SwipeItems x:Key="left" 
+												Mode="Reveal">
+										<SwipeItem Text="Reply All" 
+												   IconSource="{StaticResource ReplyAllIcon}"
+												   Background="#FF3e6fa7" 
+												   Foreground="White"/>
+										<SwipeItem Text="Open" 
+												   IconSource="{StaticResource ReadIcon}"
+												   Background="#FFff9501" 
+												   Foreground="White"/>
+									</SwipeItems>
+									<SwipeItems x:Key="right" 
+												Mode="Execute">
+										<SwipeItem Text="Delete" 
+												   IconSource="{StaticResource DeleteIcon}"
+												   Background="#FFF4B183" 
+												   Invoked="DeleteItem_ItemInvoked"/>
+									</SwipeItems>
+								</ListView.Resources>
+
+								<ListView.ItemTemplate>
+									<DataTemplate>
+										<SwipeControl BorderThickness="0,1,0,0" 
+													  BorderBrush="{ThemeResource ButtonBackground}" 
+													  Height="68"
+													  LeftItems="{StaticResource left}"
+													  RightItems="{StaticResource right}">
+											<TextBlock Text="{Binding}"
+													   FontSize="24" 
+													   Margin="12"
+													   HorizontalAlignment="Stretch" 
+													   VerticalAlignment="Center"/>
+										</SwipeControl>
+									</DataTemplate>
+								</ListView.ItemTemplate>
+							</ListView>
+						</smtx:XamlDisplay>
+
+						<!-- Swipe with gradient background -->
+						<smtx:XamlDisplay UniqueKey="SwipeControlSamplePage_FluentGradientSwipe"
+										  smtx:XamlDisplayExtensions.Header="SwipeItem with gradient background">
+							<Border>
+								<Border.Resources>
+									<FontIconSource x:Key="LockIcon" 
+													Glyph="&#xE107;"/>
+									<LinearGradientBrush x:Key="PurpleGradient" 
+														 StartPoint="0,0.5" 
+														 EndPoint="1,0.5">
+										<GradientStop Color="#ff8990f9" 
+													  Offset="0.0"/>
+										<GradientStop Color="#ff5b66fb" 
+													  Offset="0.5"/>
+										<GradientStop Color="#ff5c1df4" 
+													  Offset="1.0"/>
+									</LinearGradientBrush>
+										<SwipeItems x:Key="right" 
+													Mode="Execute">
+											<SwipeItem Text="Lock" 
+													   Background="{StaticResource PurpleGradient}"
+													   BehaviorOnInvoked="Close" 
+													   IconSource="{StaticResource LockIcon}"/>
+										</SwipeItems>
+								</Border.Resources>
+								<SwipeControl BorderThickness="1" 
+											  BorderBrush="{ThemeResource ButtonBackground}"
+											  RightItems="{StaticResource right}"
+											  Margin="12" 
+											  Height="68">
+									<TextBlock Text="Swipe Left" 
+											   Margin="12"
+											   HorizontalAlignment="Center" 
+											   VerticalAlignment="Center"/>
+								</SwipeControl>
+							</Border>
+						</smtx:XamlDisplay>
+					</StackPanel>
+				</DataTemplate>
+			</local:SamplePageLayout.FluentTemplate>
+
+			<local:SamplePageLayout.CupertinoTemplate>
```

Comment:
Remove the `<CupertinoTemplate />` block if the control doesnt show case a style for the said theme, otherwise we get an empty "Cupertino" tab with nothing in it.  And, we should probably do the same for other existing samples in uno.themes too: bottom-nav, chip, commandbar, etc.

Reply:
Yes good suggestion @Xiaoy312, I will make sure to do the same on existing samples when I'm working on them

Reply:
@lukeblevins Can you please rebase this PR and make this adjustment? Will merge after